app/src/main/java/ch/hikemate/app/ui/navigation/BottomBarNavigation.kt

```diff
@@ -0,0 +1,63 @@
+package ch.hikemate.app.ui.navigation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.semantics.SemanticsPropertyKey
+import androidx.compose.ui.semantics.semantics
+
+const val TEST_TAG_BOTTOM_BAR = "TEST_TAG_BOTTOM_BAR"
+const val TEST_TAG_MENU_ITEM_PREFIX = "TEST_TAG_MENU_ITEM_"
+
+val IsSelectedKey = SemanticsPropertyKey<Boolean>("IsSelected")
```

Comment:
Question: What is a SemanticsPropertyKey? From what I read, it's something related to tests. Is it so? If so, could you write a small comment, please?

Reply:
I removed it. Forgot to do it.